web3.js/src/connection.ts

```diff
@@ -2054,9 +2111,24 @@ export class Connection {
    */
   async getProgramAccounts(
     programId: PublicKey,
-    commitment?: Commitment,
+    config?: GetProgramAccountsConfig,
```

Comment:
Could you make this backwards compatible? The type can be `GetProgramAccountsConfig | Commitment`